app/services/attachments/create_service.rb

```diff
@@ -40,6 +42,17 @@ def execute # rubocop:disable Metrics/CyclomaticComplexity, Metrics/AbcSize, Met
 
       @attachments.each(&:save)
 
+      if @attachable.instance_of?(Sample) && @include_activity
+        @attachable.project.namespace.create_activity key: 'namespaces_project_namespace.samples.attachment.create',
+                                                      owner: current_user,
+                                                      trackable_id: @attachable.id,
+                                                      parameters: {
+                                                        sample_name: @attachable.puid,
```

Comment:
**nitpick (blocking):** Can you change this parameter to `sample_puid`

Reply:
Updated in 032cb1c08ee993dee5d18b74ef198b03bafe07b3